android/examples/Example_Application/app/build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'com.android.application'
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
+    ndkVersion "21.3.6528147"
```

Comment:
This is the same NDK version as `ironoxide-android` for consistency.

Reply:
Good call. It wouldn't hurt to put a comment in here pointing that out. It'd be nice to keep them in sync in the future.

Reply:
Sounds good. Done